### PR TITLE
Attributes in Common

### LINKED ISSUE #40: Attributes in Common
image

### PR DESCRIPTION
**Pull request by rolikaagarwal**
Closes #40 

FULL SCREEN- 

image

REDUCED SCREEN SIZE- 

image

MOBILE SCREEN--

image

### CODE REVIEW

**Comment by AbhiPatel10** [comment]
Remove Max width

**Comment by rolikaagarwal** [comment]
Removed max-width.

File: index.html [hunk 1]
```diff
@@ -230,6 +230,26 @@ <h1>Build With Quality Materials</h1>
...
+        <h4><i class="fa-regular fa-circle-dot"></i>Efficient Worker</h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Resilient</h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Reliable</h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Articulate</h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Good Memory Recall </h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Various Educational Backgrounds</h4>
+        <h4><i class="fa-regular fa-circle-dot"></i>Contruction Management, Civil Engineer, Architecture, Interior Design</h4>
+      </div>
+    </div>
+    <!-- ATTRIBUTES IN COMMON SECTION ENDS -->
```
Comment (AbhiPatel10) [inline index.html:252]: This is not a good way to write HTML code, Use unordered list (ul  --- li)

**Comment by rolikaagarwal** [comment]
changed HTML code.

**Comment by AbhiPatel10** [comment]
h4 is used for headings, so this is the list, and here h4 cannot be used. Use CSS to give style.

**Pull request merged**